### PR TITLE
Handling namespaces/tables/columns with reserved keywords in Cosmos DB adapter

### PR DESCRIPTION
**Pull request by thongdk8**
Please take a look! Thank you!

### CODE REVIEW

File: core/src/main/java/com/scalar/db/storage/cosmos/CosmosAdmin.java [hunk 1]
```diff
@@ -306,7 +306,7 @@ public void truncateTable(String namespace, String table) throws ExecutionExcept
 
       CosmosPagedIterable<Record> records =
           container.queryItems(
-              "SELECT t." + ID + ", t." + CONCATENATED_PARTITION_KEY + " FROM " + table + " t",
+              "SELECT t." + ID + ", t." + CONCATENATED_PARTITION_KEY + " FROM " + " t",
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/storage/cosmos/CosmosAdmin.java:309]:
```suggestion
              "SELECT t." + ID + ", t." + CONCATENATED_PARTITION_KEY + " FROM " + "t",
```

File: core/src/main/java/com/scalar/db/storage/cosmos/CosmosUtils.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.scalar.db.storage.cosmos;
+
+public class CosmosUtils {
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/storage/cosmos/CosmosUtils.java:3]:
```suggestion
public final class CosmosUtils {

  private CosmosUtils() {}

```

**Review by brfrn169 — COMMENTED** [review]
Left very minor suggestions. Other than that, LGTM. Thank you.

File: core/src/main/java/com/scalar/db/storage/cosmos/CosmosUtils.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.scalar.db.storage.cosmos;
+
+public final class CosmosUtils {
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/storage/cosmos/CosmosUtils.java:3]: Please don't just ignore my suggestion. If you have reasons why you don't want to apply my suggestion, please tell me and we can discuss it:
```suggestion
public final class CosmosUtils {

  private CosmosUtils() {}

```

Comment (thongdk8) [reply core/src/main/java/com/scalar/db/storage/cosmos/CosmosUtils.java:3]: Ah sorry, I missed the method.

Comment (brfrn169) [reply core/src/main/java/com/scalar/db/storage/cosmos/CosmosUtils.java:3]: > Why do we need this method btw?

I found a good article for it. You can check it:
https://www.informit.com/articles/article.aspx?p=1216151&seqNum=4

Comment (thongdk8) [reply core/src/main/java/com/scalar/db/storage/cosmos/CosmosUtils.java:3]: Great! Thank you!

File: core/src/integration-test/java/com/scalar/db/storage/cosmos/CosmosWithReservedKeywordIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
+package com.scalar.db.storage.cosmos;
+
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.storage.StorageWithReservedKeywordIntegrationTestBase;
+import java.util.Map;
+import java.util.Optional;
+
+public class CosmosWithReservedKeywordIntegrationTest
```
Comment (feeblefakie) [inline core/src/integration-test/java/com/scalar/db/storage/cosmos/CosmosWithReservedKeywordIntegrationTest.java:8]: I don't fully get this but this class seems not referenced at all. Is it OK?

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: core/src/integration-test/java/com/scalar/db/storage/cosmos/CosmosWithReservedKeywordIntegrationTest.java (hunk 4 above)
Comment (thongdk8) [reply core/src/integration-test/java/com/scalar/db/storage/cosmos/CosmosWithReservedKeywordIntegrationTest.java:8]: Seems the IDE still loading? There is no warning about that in my IDE. The CI running included this class already I think

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**